Runtime/Scripts/MeshSyncServer.cs

```diff
@@ -455,14 +478,24 @@ public void StartServer()
 
         // bones & blendshapes are handled by CaptureSkinnedMeshRenderer()
     }
-    #endregion //ServeScene
+        #endregion //ServeScene
 
 
-    #region Events
+        #region Events
 
 #if UNITY_EDITOR
-    void OnValidate() {
+    void OnValidate()
+    {
         CheckParamsUpdated();
+
+        if (m_DCCAsset != null)
+        {
+            var path = AssetDatabase.GetAssetPath(m_DCCAsset);
+            if (System.IO.Path.GetExtension(path) != ".blend")
```

Comment:
Are we assuming that we only handle Blender ?
This is not the case as I am sure you are already aware of.

Reply:
Yeah good point, I've refactored this so it doesn't check the filetype anymore.